rosbag2_compression/src/rosbag2_compression/sequential_compression_writer.cpp

```diff
@@ -104,7 +106,8 @@ void SequentialCompressionWriter::compression_thread_fn()
       {
         // Now that the message is compressed, it can be written to file using the
         // normal method.
-        std::lock_guard<std::recursive_mutex> storage_lock(storage_mutex_);
+        std::lock_guard<std::recursive_mutex> storage_lock(
+          storage_mutex_);
```

Comment:
```suggestion
        std::lock_guard<std::recursive_mutex> storage_lock(storage_mutex_);
```

revert unrelated formatting change